app/domains/schulden/vorabcheck/context.ts

```diff
@@ -1,8 +1,20 @@
 import { z } from "zod";
-import { YesNoMaybeAnswer } from "~/services/validation/YesNoAnswer";
+import { YesNoAnswer } from "~/services/validation/YesNoAnswer";
 export const context = {
-  hasKontopfaendung: YesNoMaybeAnswer,
-  hasPKonto: YesNoMaybeAnswer,
+  hasKontopfaendung: YesNoAnswer,
+  hasPKonto: YesNoAnswer,
+  schuldenBei: z.enum([
+    "privatpersonen",
+    "behörden",
+    "nicht-zurückzahlen",
+    "finanzamt",
+    "beitragsservice",
+    "unterhaltsschulden",
+    "strafe-nicht-zahlen",
+    "rechnungen-nicht-bezahlen",
+    "glaeubigerin-unbekannt",
+  ]),
```

Comment:
tricky one but here is somewhere I'd actually invoke an [english translation](https://digitalservicebund.atlassian.net/wiki/spaces/DIGITALSER/pages/1509425158/Language+in+Code+using+German+or+English+words#Our-Solution%3A-The-Hybrid-Approach) (not only to get rid of the umlauts in the enum, which I think may cause problems if we try to infer its type, but also for clarity). I think we should keep the proper, more-or-less untranslateable nouns (like Finanzamt, Beitragsservice), and translate the rest

Also, I think we're missing 3 options

Reply:
**English Translation:** 
- i think the english translation will be even more confusing here also i think it's a domain specific. 
- so it maps perfectly to the design and provide more consistency.
- so if there is no severe issue i would like to keep it in german
**Missing options:** 
the design is being updated and not finalized yet. (it will be a constant update until design team finishes)

Reply:
i think there may be a misunderstanding regarding "domain specific"; my understanding was as follows:
* terms like "Kontopfändung" or "pKonto" are "domain-specific" in the sense that their only possible context is in the domain, and translating them would make no sense
* terms like "nicht-zurückzahlen" (which, after looking at the Figma, should probably be more explicit as "Kredit": _what exactly_ isn't zurück-zahl-able?) are both general and easily exchanged with their english equivalents -- in this case, 'credit'

though i could be misunderstanding what @frederike-ramin meant, and would be curious to hear her thoughts.

Also, after looking at the Figma, it seems as if the design itself (i.e. number of options) is finalized and "Ready for Dev", but the designers are still working on the copy, unless there's some context I'm missing?

Reply:
overall, i would suggest
```suggestion
  debtSource: z.enum([
    "private",
    "authorities",
    "credit",
    "finanzamt",
    "beitragsservice",
    "supportPayments",
    "fine",
    "unpaidBill",
    "unknown",
  ]),
```

Reply:
I'm almost regretting this language-in-code thing a bit. The main purpose was: "Don't overthink it."
I personally am leaning towards all-German words. But that's just personal preference. I think both options are fine. I would probably stick to "schulden" (as we also use it in the url) and the "kredit" distinction seems important in contrast to "nicht-zurückzahlen" semantically but otherwise, I wouldn't overthink it.

Reply:
okay then we stick with German and schulden for now. 
 updating `nicht-zurückzahlen`.